Examples/VoiceMemos/VoiceMemos/VoiceMemos.swift

```diff
@@ -262,16 +271,14 @@ let voiceMemosReducer = Reducer<VoiceMemosState, VoiceMemosAction, VoiceMemosEnv
       }
       return .none
 
+    case let .voiceMemo(index: index, action: .remove):
```

Comment:
can now listen for the `.remove` action in the voice memos row to handle removing it from state.